### PR TITLE
Updating manifest for signed builds.

### PR DESCRIPTION
**Pull request by anpaz**

### CODE REVIEW

File: build/manifest.ps1 [hunk 1]
```diff
@@ -13,7 +27,8 @@
         "Microsoft.Quantum.Runtime.Core",
         "Microsoft.Quantum.Simulators",
         "Microsoft.Quantum.Xunit"
-    );
+    ) | ForEach-Object { Join-Path $Env:NUGET_OUTDIR "$_.$Env:NUGET_VERSION.nupkg" };
```
Comment (rmshaffer) [inline build/manifest.ps1:30]: Do the `.snupkg` files also need to be signed, if they exist? Should they be added here somehow? (This comment also applies to similar changes across all the repos.)

**Review by bettinaheim — APPROVED** [review]
Seems fine, aside from the build failure that still needs to be resolved.

File: build/manifest.ps1 (hunk 1 above)
Comment (anpaz) [reply build/manifest.ps1:30]: I think they do. Practically it doesn't matter as for nuget packages, as opposed to dlls, we submit for signing everything that we found in the NUGET_OUTDIR folder (which includes the `.snupkg`s). So, this is not really needed for signing.
We do use it to validate that the package is created on local builds. So for now, I'm just going to leave it like this.

**Pull request merged**